src/controllers/libraryController.ts

```diff
@@ -31,36 +31,34 @@ export class LibraryController implements Disposable {
         this.disposable.dispose();
     }
 
-    public async addLibraries(libraryGlobs?: string[]) {
-        if (!libraryGlobs) {
-            libraryGlobs = [];
-            const workspaceFolder: WorkspaceFolder | undefined = Utility.getDefaultWorkspaceFolder();
-            const isWindows = process.platform.indexOf("win") === 0;
-            const results: Uri[] | undefined = await window.showOpenDialog({
-                defaultUri: workspaceFolder && workspaceFolder.uri,
-                canSelectFiles: true,
-                canSelectFolders: isWindows ? false : true,
-                canSelectMany: true,
-                openLabel: isWindows ? "Select jar files" : "Select jar files or directories",
-                filters: { Library: ["jar"] },
-            });
-            if (!results) {
-                return;
-            }
-            libraryGlobs = await Promise.all(results.map(async (uri: Uri) => {
-                // keep the param: `includeWorkspaceFolder` to false here
-                // since the multi-root is not supported well for invisible projects
-                const uriPath = workspace.asRelativePath(uri, false);
-                return (await fse.stat(uri.fsPath)).isDirectory() ? `${uriPath}/**/*.jar` : uriPath;
-            }));
-        }
-
+    public async addLibraryGlobs(libraryGlobs: string[]) {
         const setting = Settings.referencedLibraries();
         setting.exclude = this.dedupAlreadyCoveredPattern(libraryGlobs, ...setting.exclude);
         setting.include = this.updatePatternArray(setting.include, ...libraryGlobs);
         Settings.updateReferencedLibraries(setting);
     }
 
+    public async addLibraries(folders?: boolean) {
```

Comment:
Change the param name to `canSelectFolders` to just align with the API?

Reply:
Makes sense, fixed in a735dc5.